uber/utils.py

```diff
@@ -14,8 +14,6 @@ def __init__(self, page, *args, **kwargs):
         args = [self.quote(s) for s in args]
         kwargs = {k: self.quote(v) for k, v in kwargs.items()}
         cherrypy.HTTPRedirect.__init__(self, page.format(*args, **kwargs))
```

Comment:
Hmm, I don't remember offhand why this if statement was here.  Are we 100% sure that removing it isn't going to cause a problem?

My vague recollection is that since CherryPy itself is speaking HTTP, we had to replace that with HTTPS since we're behind Apache.  Or something like that, I don't really recall.  (This is a good reason to get better about writing docstrings, haha.)